### PR TITLE
ENH: Added skipcols option for CParser

### PR DESCRIPTION
**Pull request by terrytangyuan**
See #10882 
But I am having trouble testing this change. When I run `nosetests pandas/io/tests/test_parsers.py`, it says `Segmentation fault: 11` and this pops up `Python quit unexpectedly while using the parser.so plug-in.`

Any ideas of how to deal with this?

### CODE REVIEW

**Comment by terrytangyuan** [comment]
@jreback Do you know how to fix this? Thanks.

**Comment by jreback** [comment]
you need to debug what's happening in cython prob. I usually just insert print statements, but you can also use ``gdb`` (though haven't done it in a while)

**Comment by terrytangyuan** [comment]
Ok. Thanks. I thought it's a problem of not having installed some neccessary dependencies.

File: pandas/io/parsers.py [hunk 1]
```diff
@@ -364,6 +367,7 @@ def parser_f(filepath_or_buffer,
                  converters=None,
                  dtype=None,
                  usecols=None,
+                 skipcols=None,
```
Comment (terrytangyuan) [inline pandas/io/parsers.py]: @jreback It looks like after I add this line, that error will occur. It must have something to do with the c code, right?

**Comment by jreback** [comment]
closing, pls reopen if you can fix according to comments

**Pull request closed without merging**